regimes/se/tax_categories.go

```diff
@@ -0,0 +1,94 @@
+package se
+
+import (
+	"github.com/invopop/gobl/cal"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/num"
+	"github.com/invopop/gobl/tax"
+)
+
+// Tax rates specific to Sweden.
+const (
+	TaxRateStandard       cbc.Key = "Normalskattesats"
+	TaxRateReduced        cbc.Key = "Skattesats 12 procent"
+	TaxRateHeavilyReduced cbc.Key = "Skattesats 6 procent"
+	TaxRateExempt         cbc.Key = "Momsfri"
+)
```

Comment:
Not quite sure what these are for... They'd also fail validation as `cbc.Key` has a strict format.

Reply:
Vestigial from copy-pasting from other countries' regimes. Removing